test/distributed/fsdp/test_fsdp_clip_grad_norm.py

```diff
@@ -0,0 +1,71 @@
+# Owner(s): ["oncall: distributed"]
+
+import sys
+
+import torch
+from torch import distributed as dist
+from torch.distributed.fsdp import FullyShardedDataParallel as FSDP
+from torch.nn import Linear, GELU
+from torch.testing._internal.common_distributed import skip_if_lt_x_gpu
+from torch.testing._internal.common_fsdp import (
+    FSDPTest,
+)
+from torch.testing._internal.common_utils import TEST_WITH_DEV_DBG_ASAN, run_tests
+
+
+if not dist.is_available():
+    print("Distributed not available, skipping tests", file=sys.stderr)
+    sys.exit(0)
+
+if TEST_WITH_DEV_DBG_ASAN:
+    print(
+        "Skip dev-asan as torch + multiprocessing spawn have known issues",
+        file=sys.stderr,
+    )
+    sys.exit(0)
+
+
+class TestClipGradNorm(FSDPTest):
+    class SimpleNN(torch.nn.Module):
+        def __init__(self, linear_size, rank=None):
+            super().__init__()
+            self.fc1 = Linear(*linear_size[0])
+            self.gelu = GELU()
+            self.fc2 = Linear(*linear_size[1])
+
+        def forward(self, inp):
+            return self.fc2(self.gelu(self.fc1(inp)))
+
+    @skip_if_lt_x_gpu(2)
+    def test_one_iteration(self):
+        """Test FSDP with clip grad norm."""
+        model = self.SimpleNN([[3, 15], [15, 17]])
+        group = dist.distributed_c10d._get_default_group()
+        input = torch.rand(8, 3)
+        norm_type = 2.5
+        model.to(self.rank)
+        model = FSDP(model)
+        self.assertTrue(len(input) >= self.world_size)
+        in_data = torch.Tensor(input[self.rank]).to(self.rank)
```

Comment:
nit:
`in_data = torch.Tensor(input[self.rank], device=self.rank)` is a bit more concise.